### ISSUE TITLE
How to subscribe to a web hook?

### ISSUE THREAD

**Issue by jwillmer** [emqtt/emq-web-hook#27]
Based on my knowledge about web hooks I need to subscribe to a web hook first by giving him a URL he should send the payload to. I don't see how to do this in your README.

**Comment by s4kibs4mi** [comment]
@jwillmer do changes in config file. Config file for emq-web-hook located in plugins/emq-web-hook.

**Comment by jwillmer** [comment]
Yes, I know the config. The config is also posted in the README. I am asking for how do I subscribe 1-n applications to the web hooks. I guess the reference to the config means that as of now this is a manual process that can't be done via web interface?

**Comment by s4kibs4mi** [comment]
@jwillmer Understood. Till now it works as 1-1 application. if you check config there is option to add one endpoint. You can use a middleware who will consume webhook's request and distribute among n applications.

**Comment by s4kibs4mi** [comment]
@emqplus can we use endpoint as array than string ? Then it will be possible to send request to multiple applications for an event of webhook.

**Comment by jwillmer** [comment]
I lately design my own web hook endpoint for a web application. I oriented myself on the excellent work of [pipedrive.com](https://www.pipedrive.com). Below you can see some screenshots of there web UI for creating and managing web hooks.

![pipedrive ui 1](https://user-images.githubusercontent.com/1503577/34326033-f57ed7b6-e8a3-11e7-9c7d-7deef2dc41b3.png)

![pipedrive ui 2](https://user-images.githubusercontent.com/1503577/34326034-f5b47236-e8a3-11e7-92f6-16d85d8d6db5.png)

The point is that you can subscribe to defined events. I am not interested in all events. I only like to be notified about one - maybe I even don't care about the payload instead I just do something then I get notified. This brakes if you just send notifications for all events.

**Comment by s4kibs4mi** [comment]
@jwillmer only keep the events in config file you want to get notified. Remove the events from config those are not necessary.
